politeiawww/api/v1/v1.go

```diff
@@ -710,3 +714,42 @@ type UserCommentsVotes struct{}
 type UserCommentsVotesReply struct {
 	CommentsVotes []CommentVote `json:"commentsvotes"`
 }
+
+// VoteOptionResult describes a VotingOption info
+// along with the total votes it received
```

Comment:
Make comments use < 80 cols.

Add to the comment if this is a struct or a command.